.github/workflows/check-plugin-docs.js

```diff
@@ -34,6 +35,22 @@ async function checkPluginDocs() {
           )} are defined. Additional information on this configuration can be found in "website/README.md".`
         );
       }
+      // Validate pluginTier property
+      const { pluginTier } = pluginEntry;
+      if (typeof pluginTier !== "undefined") {
```

Comment:
I've added check here to ensure that when `pluginTier` is defined, it is a valid value.